### DISCUSSION TITLE
404 on new install

### DISCUSSION THREAD

**Discussion by CardSnap** [LycheeOrg/Lychee#2260] [Q&A]
### 404 on new install
`Not Found/404 when attempting to go to url`
Running apache 2.4 on Debian 12 

### Steps to reproduce the issue
**Steps to reproduce the behavior:**
* Install staps
1.  Clone repo and set the web root to lychee/public
2.  Run `composer install --no-dev` 
3.  Run `npm install to install`
4.  Run `npm run build`
5.  Copy .env.example as .env and edited
6.  `php artisan key:generate`
7.  `php artisan migrate`
8.  `sudo chown -R www-data:www-data` on lychee dir

navigated to my the url and saw a 404 page. 

I'm missing a step related to my apache conf file for lychee

lychee.conf
```
<VirtualHost *:80>
    ServerName {same domain used in .env}

    DocumentRoot {webroot}/Lychee/public

    ErrorLog ${APACHE_LOG_DIR}/error.log
    CustomLog ${APACHE_LOG_DIR}/access.log combined
</VirtualHost>
```

Followed along with https://lycheeorg.github.io/docs/installation.html#apache and https://lycheeorg.github.io/docs/upgrade.html#using-apache as possible.

**Comment by ildyria** [comment]
No, this is not a bug, nor an issue.
**This is part of the assumed knowledge when using Lychee or any other website when setting a web server.**
[Firefly-iii](https://docs.firefly-iii.org/how-to/firefly-iii/installation/docker/) which is also based on Laravel does not even go that far, even in their [managed server steps](https://docs.firefly-iii.org/how-to/firefly-iii/installation/self-managed/).

I do agree that there might be a little room for improvement on the install steps.
If you want to make this installation smoother, please open a pull request on https://github.com/LycheeOrg/LycheeOrg.github.io with the modifications you are thinking of.

**Reply by CardSnap** [reply]
With what knowledge that we've both established I am missing am I supposed to place into a PR? Thank you very much for slamming the door in my face.

**Comment by CardSnap** [comment]
With what knowledge that we've both established I am missing am I supposed to place into a PR? Thank you very much for slamming the door in my face. Times two.

**Reply by ildyria** [reply]
You literally found what you were missing...

In `/etc/apache2/site-enabled/lychee.conf`
```
<VirtualHost *:80>
    ServerName {same domain used in .env}

    DocumentRoot {webroot}/Lychee/public

    ErrorLog ${APACHE_LOG_DIR}/error.log
    CustomLog ${APACHE_LOG_DIR}/access.log combined
</VirtualHost>
```

and in `/etc/apache2/conf-enabled/lychee.conf`
```
<Directory /{webroot}/Lychee>
	Options FollowSymLinks
	AllowOverride All
	Require all granted
</Directory>
```

**Reply by ildyria** [reply]
Furthermore I re-read the instructions, they are extremely clear. Especially https://lycheeorg.github.io/docs/upgrade.html#using-apache . So yes I am sorry but I can't help you more than that.

And please make sure you have mod rewrite enabled.
